mmengine/registry/registry.py

```diff
@@ -240,27 +255,25 @@ def switch_scope_and_registry(self, scope: str) -> Generator:
             # Get registry by scope
             if default_scope is not None:
                 scope_name = default_scope.scope_name
-                if scope_name in PKG2PROJECT:
-                    try:
-                        module = import_module(f'{scope_name}.utils')
-                        module.register_all_modules(False)  # type: ignore
```

Comment:
Will it break old downstream repositories if removing this line `module.register_all_modules` because they do not provide the locations when defining Registry.

Reply:
I add a check logic in `import_from_location` to avoid BC breaking.